content/_index.md

```diff
@@ -10,3 +10,22 @@ Welcome to the website of the NWERC 2022!
 The Northwestern Europe Regional Contest (NWERC) is a contest in which teams from universities all over the Northwestern part of Europe are served a series of algorithmic problems. The goal of each team is to solve as many problems as possible within the 5 hour time limit. Potential solutions are submitted and corrected by an automated judging system. The team that solves the most problems at the end of the contest qualifies for the ICPC World Finals.
 
 The 2022 edition of the Northwestern European Regional Contest (NWERC) will be held on the 25th of November until the 27th of November 2022. The contest will be held on the campus of the Delft University of Technology.
+
+{{% row %}}
+{{% column %}}
+
+## Contest material
+
+- [Contest Livestream](https://youtu.be/aDsW8J8P7go)
+- [Award Ceremony Livestream](https://youtu.be/nbr_v73cvfQ)
+
+{{% /column %}}
+{{% column %}}
+
+## Test session material
+
+- [Opening Ceremony Livestream](https://youtu.be/nvOgM5o3uMk)
```

Comment:
Do we consider the opening part of test session material? I think it's part of the main contest, just happens to be on a different day